cmd/kubelet/app/options/options.go

```diff
@@ -369,6 +370,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Var(&utilflag.IPVar{Val: &c.Address}, "address", "The IP address for the Kubelet to serve on (set to '0.0.0.0' or '::' for listening on all interfaces and IP address families)")
 	fs.Int32Var(&c.Port, "port", c.Port, "The port for the Kubelet to serve on.")
 	fs.Int32Var(&c.ReadOnlyPort, "read-only-port", c.ReadOnlyPort, "The read-only port for the Kubelet to serve on with no authentication/authorization (set to 0 to disable)")
+	fs.BoolVar(&c.FailCgroupV1, "fail-cgroupv1", c.FailCgroupV1, "Prevent the kubelet from starting on the host using cgroup v1.")
```

Comment:
nit: s/on the host/on hosts/

Reply:
Since kubelet acts on a single node, I see that other options in that file also use the singular reference. e.g an option to fail kubelet when swap is on, https://github.com/kubernetes/kubernetes/blob/1854839ff08eba7434ebfa98ab84de2d3a62d4ce/cmd/kubelet/app/options/options.go#L362

`Makes the Kubelet fail to start if swap is enabled on the node`